src/System.Memory/tests/Span/GcReporting.cs

```diff
@@ -12,6 +12,7 @@ public static class SpanGcReportingTests
         /// This is a simple sanity test to check that GC reporting for Span is not completely broken, it is not meant to be
         /// comprehensive.
         /// </summary>
+        [Theory]
```

Comment:
any idea why this wasn't caught with the existing enabled analyzers? I thought I fixed all of these.

Reply:
Don't know.  This was triggering https://xunit.net/xunit.analyzers/rules/xUnit1008, which was disabled.

Reply:
Super weird. Means that the analyzers are not guaranteed to be correct.

Reply:
Looking at the xunit analyzer code for 1013, it only fires if the public method in question is on a class with other tests (a reasonable heuristic)... this class doesn't have any others.